clients/drcachesim/tools/opcode_mix.cpp

```diff
@@ -316,16 +326,27 @@ opcode_mix_t::print_results()
             for (const auto &keyvals : shard.second->category_counts) {
                 total.category_counts[keyvals.first] += keyvals.second;
             }
+            total.filetype = shard.second->filetype;
         }
     }
     std::cerr << TOOL_NAME << " results:\n";
     std::cerr << std::setw(15) << total.instr_count << " : total executed instructions\n";
     std::vector<std::pair<int, int64_t>> sorted(total.opcode_counts.begin(),
                                                 total.opcode_counts.end());
     std::sort(sorted.begin(), sorted.end(), cmp_val);
-    for (const auto &keyvals : sorted) {
-        std::cerr << std::setw(15) << keyvals.second << " : " << std::setw(9)
-                  << decode_opcode_name(keyvals.first) << "\n";
```

Comment:
Similarly to https://github.com/DynamoRIO/dynamorio/pull/6663/files#r1584655333 here.
Should we add a name to OP_INVALID (is the failure on decode_opcode_name(OP_INVALID) intended?) or work around it as I did here?

Reply:
We should make it work.  See the OP_UNDECODED suggestion.  Use OP_nop??  No; I would still think making OP_UNDECODED work seems best.

Reply:
OP_UNDECODED seems to work. Done.
The only change I had to make to the opcode_mix analyzer was changing the ISA to DR_ISA_REGDEPS id the filetype of the trace is regdeps.